supervision/__init__.py

```diff
@@ -1,5 +1,6 @@
 __version__ = "0.9.0"
 
+from supervision.classification.core import ClassificationDataset, Classifications
```

Comment:
This import is dead. This package does not exist.

Reply:
`Classifications` class should be in `supervision.classification.core`, but `ClassificationDataset` class should be in `supervision.dataset.core`